### ISSUE TITLE
console is garbled when booting with spice graphics

### ISSUE THREAD

**Issue by dweomer** [rancher/k3os#306]
**Version**
v0.5.0 and greater

**Architecture**
x86_64

**Describe the bug**
Garbled console when using spice/qxl: 
![k3os-0 7 1](https://user-images.githubusercontent.com/1033444/69750232-827f9a80-1109-11ea-8867-de177563c557.png)

**To Reproduce**
Run Qemu with Spice graphics:
- https://rancher-users.slack.com/archives/CJ75F6WJ3/p1574874763383200
- #245 

**Expected behavior**
The console should not be garbled.

**Actual behavior**
The console is garbled

**Additional context**
n/a

**Comment by pagong** [comment]
Easy solution for KVM/QEMU screen issue:
I've switched the gfx card from spice/qxl to **spice/vga** and the console does not get garbled anymore. 
This works for k3os v0.5.0, v0.6.0 and v0.7.1.

**Comment by liyimeng** [comment]
I also similar issue. Changing the video driver as pagong suggested help. It seems works for all other driver except default one.

**Comment by stove-panini** [comment]
Unfortunately, if you're using oVirt as your virtualization platform, VGA+SPICE isn't an option :(

**Comment by stablecube** [comment]
Yeah this is a major PITA. In a lot of situations it is not an option to change the graphics used (DigitalOcean console).

**Comment by dweomer** [comment]
Looks like we are completely lacking in qxl driver support as per the upstream ubuntu kernel. I'll get this into the k3os-kernel and update master soonish.

**Comment by dweomer** [comment]
@pagong @liyimeng @stove-panini @stablecube have you tried forcing the `qxl` kernel module to load (or other module relevant to your virtualized rendering setup) via a `k3os.module=qxl` kernel parameter or the `config.yaml` below:

**Comment by pagong** [comment]
Sorry, I've needed console access only to get k3os installed on disk on KVM.
Switching to VGA (from QXL) solved this for me.
After these very first steps I only use ssh to login to k3os.

If there's a way to persist the installation without using the QEMU console I'd be very happy to try it out.
